vendor/licenses/grammar/language-sass.txt

```diff
@@ -0,0 +1,53 @@
+---
+type: grammar
+name: language-sass
+version: d07f9e7bebac73a2c6a0dceae4c099ea6d60fd4c
+license: other
```

Comment:
> Looks like CI is failing due to the license. It looks like it's just two MIT licenses concatenated together, but it gets detected as "other".

Yeah, these Atom/Textmate licenses always trip things up and need manual reviewing. Consider this my review for the license and feel free to change "other" to "mit".

Reply:
Done, thanks for the review @lildude.